packages/core/src/shader/ShaderPool.ts

```diff
@@ -29,19 +31,22 @@ import { ShaderPass } from "./ShaderPass";
  */
 export class ShaderPool {
   static init(): void {
-    const shadowCasterPassTags = {
+    const depthOnlyPass = new ShaderPass(depthOnlyVs, depthOnlyFs, {
+      pipelineStage: PipelineStage.DepthOnly
+    });
+    const shadowCasterPass = new ShaderPass(shadowMapVs, shadowMapFs, {
       pipelineStage: PipelineStage.ShadowCaster
-    };
+    });
+    const basePasses = [depthOnlyPass, shadowCasterPass];
```

Comment:
use [shadowCasterPass, depthOnlyPass]  to be compatible with previous usage

Reply:
done